client/src/reducers/podcast-reducer.js

```diff
@@ -10,12 +10,10 @@ export default function podcastReducer(podcasts = init, action) {
       console.log(action, 'add');
       return [action.payload, ...podcasts]
     case FETCH_DESCRIPTION_SUCCESS:
-      // console.log('what?', action.podcast,action.description.data);
-      // podcasts = action.description.data.data;
-      // return  [action.payload.data.data, ...podcasts];
-      return action.payload.data.data;
-    // case 'FETCH_USER_SUBSCRIPTION_SUCCESS':
-    
+      return  [...podcasts, ...action.payload.data.data]; // double spread operators do the trick
```

Comment:
This did the trick, officially we don't change or mutate state. We instead make copies of existing and override them.